sdk/js/src/utils/consts.ts

```diff
@@ -38,6 +38,7 @@ export const CHAINS = {
   linea: 38,
   berachain: 39,
   seievm: 40,
+  snaxchain: 43,
```

Comment:
We are still updating the old sdk?  Is there some coupling to sdkV2 to be had?

Reply:
Maybe someone, ahem, you, could take out a PR in the wormhole-sdk-ts repo?

Reply:
I assume we still want to support the old sdk.

I am currently investigate how to (who should??) update sdk v2. That will obviously be a separate PR anyway, so I'm resolving this issue.